--- mod_ruid2-0.9.4/mod_ruid2.c~	2011-02-24 15:25:28.000000000 +0100
+++ mod_ruid2-0.9.4/mod_ruid2.c	2011-03-10 17:37:27.670490000 +0100
@@ -47,10 +47,10 @@
 #define MODULE_NAME		"mod_ruid2"
 #define MODULE_VERSION		"0.9.4"
 
-#define RUID_DEFAULT_UID	48
-#define RUID_DEFAULT_GID	48
-#define RUID_MIN_UID		100
-#define RUID_MIN_GID		100
+#define RUID_DEFAULT_UID	51
+#define RUID_DEFAULT_GID	51
+#define RUID_MIN_UID		50
+#define RUID_MIN_GID		50
 
 #define RUID_MAXGROUPS		4
 
